corva/event/loader/base.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import annotations
+
+import json
+from abc import abstractmethod, ABC
+
+from corva.constants import EVENT_TYPE
+from corva.event.event import Event
+
+
+class BaseLoader(ABC):
+    @abstractmethod
+    def load(self, event: str) -> Event:
+        pass
+
+    @staticmethod
+    def _load_json(event: str) -> EVENT_TYPE:
+        if not event:
+            raise ValueError('Empty event')
+
+        if not isinstance(event, str):
+            raise TypeError(f'Unknown event type {type(event)}')
```

Comment:
`json.loads` already throws an error if it cannot convert a string to json. Let's not double-check it